al2.pkr.hcl

```diff
@@ -1,10 +1,12 @@
 locals {
-  ami_name_al2 = "${var.ami_name_prefix_al2}-hvm-2.0.${var.ami_version}-x86_64-ebs"
+  ami_name_al2 = "kontain-ecs-${var.ami_name_prefix_al2}-hvm-2.0.${var.ami_version}-x86_64-ebs"
+  km_build = "../../../build"
+  target_tmp = "/tmp/"
 }
 
 source "amazon-ebs" "al2" {
   ami_name        = "${local.ami_name_al2}"
-  ami_description = "Amazon Linux AMI 2.0.${var.ami_version} x86_64 ECS HVM GP2"
+  ami_description = "Kontain Amazon Linux AMI 2.0.${var.ami_version} x86_64 ECS HVM GP2"
   instance_type   = "c5.large"
```

Comment:
In the `main` branch, there were changes with the instance types. Did you investigate those? They seemed kind of cool to me because they used things like `var.general_purpose_instance_types` instead of hard coded ec2 instance names.

Reply:
Yeah, I saw that, but it looks like they use spot instances there. Which to me doesn't look right for us.